DotNetHowToSynonyms/DotNetHowToSynonyms/Program.cs

```diff
@@ -58,13 +56,13 @@ private static SearchServiceClient CreateSearchServiceClient()
         private static SearchIndexClient CreateSearchIndexClient()
         {
             string searchServiceName = ConfigurationManager.AppSettings["SearchServiceName"];
-            string queryApiKey = ConfigurationManager.AppSettings["SearchServiceQueryApiKey"];
+            string queryApiKey = ConfigurationManager.AppSettings["SearchServiceApiKey"];
```

Comment:
We should always encourage good security practices -- in this case, the principle of least privilege. Please keep the query key and admin key separate.